hazelcast-documentation/src/HazelcastSimulator/ExecutingASimulatorTest.md

```diff
@@ -1,185 +1,310 @@
 
-
 ## Executing a Simulator Test
 
 After you install and prepare the Hazelcast Simulator for your environment, it is time to perform a test.
 
-The following steps execute a Hazelcast Simulator test.
-
-1. Install the Hazelcast Simulator.
-2. Create a folder85 for your tests. Let's call it your working folder.
-3. Copy the `simulator.properties` file from the `/conf` folder of the Hazelcast Simulator to your working folder.
-4. Edit the `simulator.properties` file according to your needs.
-5. Copy the `test.properties` file from the `/simulator-tests` folder of Hazelcast Simulator to your working folder.
-6. Edit the `test.properties` file according to your needs.
-5. Execute the `run.sh` script while you are in your working folder to perform your Simulator test.
+In the following sections, you are going to learn how to run a test from the scratch.
 
-In the following sections, we provide an example test and its output along with the required edits to the files `simulator.properties` and `test.properties`.
+### Creating and editing properties file
 
-### An Example Simulator Test
+You need to create `test.properties` file in the working directory. This command helps you to create and edit this file.
+	
+	cat > test.properties
```

Comment:
Can't this be done in a single command?

Now the user still needs to open the file using an editor.